From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Calvin Watford <watfordcalvin@gmail.com>
Date: Mon, 9 Oct 2023 23:40:42 -0600
Subject: temp: prevent unused function error

For some reason crrev.com/c/4881761 added a few functions to a delegate
in preparation for an upcoming feature... but these functions are
currently unused and trip the `-Werror,-Wunused-functions` lint rule.

Once these functions are used, this patch can be removed.

diff --git a/chrome/browser/devtools/devtools_embedder_message_dispatcher.cc b/chrome/browser/devtools/devtools_embedder_message_dispatcher.cc
index 55ed277ad1b46fb46ec378d6f2ab4825bb3a60be..c3c5581efc41691ae2eb4eb806bc40954b036282 100644
--- a/chrome/browser/devtools/devtools_embedder_message_dispatcher.cc
+++ b/chrome/browser/devtools/devtools_embedder_message_dispatcher.cc
@@ -74,6 +74,7 @@ bool GetValue(const base::Value& value, RegisterOptions* options) {
   return true;
 }
 
+#if 0
 bool GetValue(const base::Value& value, ImpressionEvent* event) {
   if (!value.is_dict()) {
     return false;
@@ -164,6 +165,7 @@ bool GetValue(const base::Value& value, KeyDownEvent* event) {
   }
   return true;
 }
+#endif  // 0
 
 template <typename T>
 struct StorageTraits {
